### PR TITLE
Add local migration script

### PR DESCRIPTION
**Pull request by olivergrabinski**
Adding scripts, docker-compose files to perform local migration from backup.

Also updated:
* Permissions `entityType`/`label`
* `enrichJsonFileEvent` function to fix a decoding failure
* `toIgnore` fix & tests added

### CODE REVIEW

File: delta/migration/scripts/cassandra-init/init.sh [hunk 1]
```diff
@@ -0,0 +1,5 @@
+until cqlsh -f /cassandra-init/cassandra_1_5.ddl && cqlsh -f /cassandra-init/create-materialized-view.ddl && sstableloader --nodes localhost  /cassandra-backups/cassandra-0/delta_1_5/messages && sstableloader --nodes localhost  /cassandra-backups/cassandra-1/delta_1_5/messages && sstableloader --nodes localhost  /cassandra-backups/cassandra-2/delta_1_5/messages; do
```
Comment (imsdu) [inline delta/migration/scripts/cassandra-init/init.sh:1]: cassandra-0, cassandra-1 and cassandra-2 have the same messages, no need to load them all

File: delta/plugins/composite-views/src/main/scala/ch/epfl/bluebrain/nexus/delta/plugins/compositeviews/model/CompositeViewProjection.scala [hunk 2]
```diff
@@ -117,7 +119,7 @@ object CompositeViewProjection {
       resourceTag: Option[UserTag],
       includeMetadata: Boolean,
       includeDeprecated: Boolean,
-      includeContext: Boolean,
+      includeContext: Boolean = false,
```
Comment (imsdu) [inline delta/plugins/composite-views/src/main/scala/ch/epfl/bluebrain/nexus/delta/plugins/compositeviews/model/CompositeViewProjection.scala:122]: There should be a better way to handle this

File: delta/migration/scripts/cassandra-init/init.sh (hunk 1 above)
Comment (olivergrabinski) [reply delta/migration/scripts/cassandra-init/init.sh]: Fixed

File: delta/plugins/composite-views/src/main/scala/ch/epfl/bluebrain/nexus/delta/plugins/compositeviews/model/CompositeViewProjection.scala (hunk 2 above)
Comment (olivergrabinski) [reply delta/plugins/composite-views/src/main/scala/ch/epfl/bluebrain/nexus/delta/plugins/compositeviews/model/CompositeViewProjection.scala]: I ended up reverting the decoder changes and writing the function to enrich the `CompositeViewEvents`.

**Review by imsdu — APPROVED** [review]

**Pull request merged**